packages/vapor/scss/tables/table.scss

```
@@ -101,7 +101,10 @@
             opacity: 0.4;
         }
 
-        &:not(.no-hover):not(.collapsible-row):not(.empty):hover td {
+        &:not(.no-hover):not(.collapsible-row):not(.empty):hover {
+            td {
+                background-color: $light-grey;
+            }
             background-color: $light-grey;
         }
```

It will be easier to add other rules if need if you swith the declaration & nested declaration
```suggestion
        &:not(.no-hover):not(.collapsible-row):not(.empty):hover {
            background-color: $light-grey;
            td {
                background-color: $light-grey;
            }
        }
```